### PR TITLE
Add ability to configure MysqlDB app via Options

### PR DESCRIPTION
**Pull request by alexvbg**
## Change Overview

This PR add ability to configure MysqlDB app via options + added function that help us configure chart values for that app.

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [x] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [x] :robot: Test

## Issues 

- fixes #issue-number

## Test Plan

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

**Review by viveksinghggits — COMMENTED** [review]
Also can you briefly explain why are we trying to do this?

File: pkg/app/mysql.go [hunk 1]
```diff
@@ -45,17 +58,51 @@ type MysqlDB struct {
...
+long_query_time=10.0
+[client]
+port=3306
+socket=/opt/bitnami/mysql/tmp/mysql.sock
+default-character-set=UTF8
+plugin_dir=/opt/bitnami/mysql/lib/plugin
+[manager]
+port=3306
+socket=/opt/bitnami/mysql/tmp/mysql.sock
+pid-file=/opt/bitnami/mysql/tmp/mysqld.pid
```
[the remark this answers is not in the document]
Comment (alexvbg) [reply pkg/app/mysql.go]: I removed it after discussion with @e-sumin. Now there is only optional parameters in MysqlDB constructor.

File: pkg/app/mysql.go [hunk 2]
```diff
@@ -42,9 +42,19 @@ type MysqlDB struct {
 	chart     helm.ChartInfo
 }
 
+type MysqlDBOptions func(*MysqlDB)
+
+func MysqlDBWithChart(chart map[string]string) MysqlDBOptions {
```
Comment (viveksinghggits) [inline pkg/app/mysql.go:47]: should this be?

```suggestion
func MysqlDBWithValues(values map[string]string) MysqlDBOptions {
```

Comment (viveksinghggits) [reply pkg/app/mysql.go:47]: or I think better wouldbe `MysqlDBHelmValues`, because we are just creating helm values for options. but it's just TILI.

Comment (viveksinghggits) [reply pkg/app/mysql.go:47]: perfect.

Comment (alexvbg) [reply pkg/app/mysql.go:47]: Fixed.

**Review by viveksinghggits — APPROVED** [review]

**Review by pavannd1 — APPROVED** [review]

**Pull request merged**